action/action_plugins/src/fault_injection_action.cpp

```diff
@@ -33,6 +33,7 @@ catch (...)
 void FaultInjectionAction::run(
   const std::shared_ptr<scenario_intersection::IntersectionManager>&)
 {
+  // TODO(yunus.caliskan): Replace the logic for ros2 or disable this action altogether.
```

Comment:
Sounds like this still needs action?

Reply:
@fred-apex-ai Oh right, this is missing. According to https://answers.ros.org/question/323329/how-to-kill-nodes-in-ros2/?answer=323332#post-id-323332 this doesn't have a direct equivalent in ROS2 unless the node in question is a lifecycle node.

@esteve @mitsudome-r  Do you have any ideas on how to resolve this issue? We could go back to the nodes this action might be targeting and make them lifecycle nodes and then use the solution from the link but a more local solution would be preferred of course.